src/GitExtensions.GerritPlugin/GitExtensions.GerritPlugin.nuspec

```diff
@@ -11,17 +11,17 @@
     <icon>Resources\IconGerrit.png</icon>
     <dependencies>
       <group>
-        <dependency id="GitExtensions.Extensibility" version="[0.1.0, 0.2.0)" />
+        <dependency id="GitExtensions.Extensibility" version="[0.2.0, 0.3.0)" />
       </group>
-      <!-- To fix Warning	NU5128	Add a dependency group for .NETFramework4.6.1 to the nuspec	GitExtensions.GerritPlugin -->
-      <group targetFramework=".NETFramework4.6.1">
-        <dependency id="GitExtensions.Extensibility" version="[0.1.0, 0.2.0)" />
+      <!-- To fix Warning NU5128 Add a dependency group for net5.0-windows7.0 to the nuspec -->
+      <group targetFramework="net5.0-windows7.0">
+        <dependency id="GitExtensions.Extensibility" version="[0.2.0, 0.3.0)" />
       </group>
     </dependencies>
   </metadata>
   <files>
     <file src="..\..\LICENSE.md" target="\" />
-    <file src="bin\$configuration$\net461\GitExtensions.GerritPlugin.dll" target="\lib\net461\" />
+    <file src="bin\$configuration$\net5.0-windows7.0\GitExtensions.GerritPlugin.dll" target="\lib\net5.0-windows7.0\" />
```

Comment:
I don't think Plugin Manager will look at this TFM. Have you tested installing the nuget?

Reply:
Hi @maraf,
Just tested - you are right. I'm able to install Gerrit plugin v2, but looks like it is not installed even if I see it's installed at Plugin manager. How can I fix that? If I switch TFM to just `net5.0-windows` without version I have an error:
```
Error	NU1012	Some dependency group TFMs are missing a platform version: net5.0-windows	
GitExtensions.GerritPlugin	C:\Program Files\dotnet\sdk\6.0.101\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets	221	
```

Looks like a target platform version is required (based on [this issue](NuGet/Home/issues/10853))?

Reply:
Would this work?
```xml
<file src="bin\$configuration$\net5.0-windows7.0\GitExtensions.GerritPlugin.dll" target="\lib\" />
```

Reply:
Yes https://github.com/gitextensions/gitextensions.pluginmanager/blob/master/src/GitExtensions.PluginManager/GitExtensions.PluginManager.nuspec#L17

Reply:
> How can I fix that? If I switch TFM to just `net5.0-windows` without version I have an error:

I have never seen that, but according the issue you mentioned and PR inside it, it seems you are doing it correctly.

Reply:
Thanks @maraf! I'll do PR to fix that.